core-web/libs/portlets/edit-ema/portlet/src/lib/services/dot-page-api.service.ts

```diff
@@ -107,7 +117,8 @@ export class DotPageApiService {
             variantName,
             experimentId,
             depth,
-            mode
+            mode,
+            publishDate: publishDate ? publishDate : undefined
```

Comment:
```suggestion
            publishDate: publishDate ?? undefined
```

Isn't this the same? 🤔

Reply:
Yup, changed!